media/quicktime_mov.ksy

```diff
@@ -126,54 +162,331 @@ types:
+      - id: modification_time
+        type: u4
+        doc: |
+          The last modification date for the media atom.  Represent the
+          calendar date and time in seconds since midnight, January 1, 1904,
+          preferably using coordinated universal time (UTC).
+      - id: time_scale
+        type: u4
+        doc: |
+          A time value that indicates the time scale for this media.
+          The number of time units that pass per second in its time
+          coordinate system.
+      - id: duration
+        type: u4
+        doc: The duration of this media in units of its time scale.
+      - id: language
         type: u2
-  fixed16:
-    doc: Fixed-point 16-bit number.
+        doc: |
+          Specifies the language code for this media. See Language code
+          values in Apple's documentation for valid language codes.
+      - id: quality
+        type: u2
+        doc: Media's playback quality.
+    instances:
+      duration_in_sec:
+        value: '(1.0 * duration) / time_scale'
+  hdlr_body:
+    doc: |
+      Declares the process by which the media data in the stream may
+      be presented, and thus, the nature of the media in a stream.
+      For example, a video handler would handle a video track.
+    doc-ref: 'https://developer.apple.com/documentation/quicktime-file-format/handler_reference_atom'
     seq:
-      - id: int_part
-        type: s1
-      - id: frac_part
+      - id: version
         type: u1
+        doc: Version of this handler information.
+      - id: flags
+        size: 3
+        doc: A 3-byte space for handler information flags.
+      - id: component_type
+        size: 4
+        doc: |
+          A four-character code that identifies the type of the handler.
+          Only two values are valid for this field - `mhlr` for media
+          handlers, and `dhlr` for data handlers.
+      - id: component_subtype
+        type: str
+        encoding: ascii
+        size: 4
+        doc: |
+          A four-character code that identifies the type of the media
+          handler or data handler. For media handlers, this field defines
+          the type of data - for example, `vide` for video data, `soun`
+          for sound data or `subt` for subtitles. For data handlers, this
+          field defines the data reference type; for example, a component
+          subtype value of `alis` identifies a file alias.
+      - id: component_manufacturer
+        size: 4
+        doc: Reserved. Set to 0.
+      - id: component_flags
+        size: 4
+        doc: Reserved. Set to 0.
+      - id: component_flags_mask
+        size: 4
+        doc: Reserved. Set to 0.
+      - id: component_name
+        size-eos: true
+        type: str
+        encoding: ascii
+        doc: |
+          A counted string that specifies the name of the component - that
+          is, the media handler used when this media was created. This
+          field may contain a zero-length (empty) string.
```

Comment:
Wow, you really did a lot of research on this (and on the entire pull request). I updated the comment, I agree that since perhaps we don't need to parse the string correctly we can leave it slightly under-implemente.

(BTW, the string at `24 "component_name":` is in Windows-1250; doesn't really matter because we can probably expect arbitrary encodings there, but I checked it out)